readme.md

```diff
@@ -1,151 +1,131 @@
+- Custom pipeline: You can create a custom pipeline with a **yml** definition that triggers **npx renovate**. [More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
 
-Get started with Renovate by checking out our [tutorial](https://github.com/renovatebot/tutorial).
+#### Run Renovate CLI
 
-### GitHub
+There are several ways to run the Renovate CLI directly.
+See docs: [Running Renovate](https://docs.renovatebot.com/getting-started/running/) for all options.
 
-We recommend that you use the Mend Renovate App.
-Install [the Mend Renovate App](https://github.com/apps/renovate) now.
-[More details on the Mend Renovate App installation](https://docs.renovatebot.com/getting-started/installing-onboarding/).
+**Targets: all platforms**
 
-### Azure DevOps
+## Docs
 
-There are two ways to run Renovate on Azure DevOps:
+### More about Renovate
 
-- Renovate Me extension
-- Custom pipeline
+- Renovate basics
+  - [Why use Renovate](https://docs.renovatebot.com/#why-use-renovate)
+  - [What does it do? / How does it work?](https://docs.renovatebot.com/key-concepts/how-renovate-works/)
+  - [Who is using it?](https://docs.renovatebot.com/#who-uses-renovate)
+- Supported platforms and languages
+  - [Supported platforms](https://docs.renovatebot.com/#supported-platforms)
+  - [Supported languages / package managers](https://docs.renovatebot.com/modules/manager/)
+- Advanced renovate usage
+  - [Accessing private packages](https://docs.renovatebot.com/getting-started/private-packages/)
+  - [Merge Confidence data](https://docs.renovatebot.com/merge-confidence/)
 
-#### Renovate Me extension
+### Renovate Docs
 
-Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
-From there you can create a pipeline with the `RenovateMe` task.
+- [Renovate Configuration](https://docs.renovatebot.com/configuration-options/)
+- [Mend Renovate Self-Hosted Docs](https://github.com/mend/renovate-ce-ee/tree/main/docs)
 
-> [!NOTE]
-> This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
+### Comparisons
 
-#### Custom pipeline
+- [Different ways to run Renovate](https://www.mend.io/renovate/)
+- [Renovate vs Dependabot Comparison](https://docs.renovatebot.com/bot-comparison/)
 
-You can create a custom pipeline with a `yml` definition that triggers `npx renovate`.
-[More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
+## Get involved
 
-### Bitbucket Cloud/Server, Forgejo, Gitea, GitLab
+### Issues and Discussions
 
-For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.
+We prefer all items to be opened as a discussion before opening an issue.
```

Comment:
```suggestion
Please open a Discussion, to get help, suggest a new feature, or to report a bug.
We only want maintainers to open Issues.
```

How about this?

Reply:
Thanks for the suggestion. I'll think about this one.
In any event, can you please remove the comma after "Discussion".